custom/icds_reports/reports/adhaar.py

```diff
@@ -158,6 +161,7 @@ def get_adhaar_sector_data(domain, config, loc_level, location_id, show_test=Fal
     }
 
 
+@quickcache(['domain', 'config', 'loc_level', 'show_test'], timeout=30 * 60)
```

Comment:
These quick cache's should be cleared when the aggregation task is finishes https://github.com/dimagi/commcare-hq/blob/master/custom/icds_reports/tasks.py#L106-L110

Reply:
(offlined)